src/demo/containers/playground/PeopleProfileDemo.tsx

```diff
@@ -13,6 +14,16 @@ export const PeopleProfileDemo = () => {
             summaryLink={"/Explore/People"}
           />
         </div>
+        <div className="row">
+          <UserCardListRotate
+            sql={"SELECT * FROM syn13897207 where isFeatured=true"}
+            count={3}
+            useQueryResultUserData={true}
+            size={MEDIUM_USER_CARD}
+            summaryLinkText={"EXPLORE ALL PEOPLE"}
+            summaryLink={"/Explore/People"}            
```

Comment:
note that you can now set the UserCard size (defaults to LARGE), and can tell it to use the query results to populate (something that the UserCardList already supports, but the UserCardListRotate needs to support for AMP-AD since not all rows contain a UserID!)